syft/rekor/utils_test.go

```diff
@@ -0,0 +1,61 @@
+package rekor
+
+import (
+	"os"
+	"testing"
+
+	"github.com/sigstore/rekor/pkg/generated/client"
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_verifyCert(t *testing.T) {
+	tests := []struct {
+		name        string
+		certFile    string
+		expectedErr string
+	}{
+		{
+			name:        "self signed cert",
+			certFile:    "test-fixtures/test-certs/self-signed-cert.pem",
+			expectedErr: "x509: certificate signed by unknown authority",
```

Comment:
if expecting a particular error, an error variable should be defined and exposed for comparison with something like https://pkg.go.dev/errors. 

However, if the test case is just saying that "this should fail" then it is sufficient to expectedErr to be a boolean and check for error to be `non-nil`

Reply:
Fixed this case, but have some questions about this in general.